enhancements/cluster-logging/cluster-logging-v2-apis.md

```diff
@@ -0,0 +1,515 @@
+see-also:
+  - "/enhancements/cluster-logging-log-forwarding.md"
+  - "/enhancements/forwarder-input-selectors.md"
+replaces: []
+superseded-by: []
+---
+
+
+# observability.openshift.io/v1 API for Log Forwarding and Log Storage
+## Summary
+
+Logging for Red Hat OpenShift has evolved since its initial release in OpenShift 3.x from an on-cluster, highly opinionated offering to a more flexible log forwarding solution that supports multiple internal (e.g LokiStack, Elasticsearch) and externally managed log storage.  Given the original components (e.g. Elasticsearch, Fluentd) have been deprecated for various reasons, this enhancement introduces the next version of the APIs in order to formally drop support for those features as well as to generally provide an API to reflect the future direction of log storage and forwarding.
+
+## Motivation
+
+### User Stories
+
+The next version of the APIs should continue to support the primary objectives of the project which are:
+
+* Collect logs from various sources and services running on a cluster
+* Normalize the logs to common format to include workload metadata (i.e. labels, namespace, name)
+* Forward logs to storage of an administrator's choosing (e.g. LokiStack)
+* Provide a Red Hat managed log storage solution
+* Provide an interface to allow users to review logs from a Red Hat managed storage solution
+
+The following user stories describe deployment scenarios to support these objectives:
+
+* As an administrator, I want to deploy a complete operator managed logging solution that includes collection, storage, and visualization so I can evaluate log records while on the cluster
+* As an administrator, I want to deploy an operator managed log collector only so that I can forward logs to an existing storage solution
+* As an administrator, I want to deploy an operator managed instance of LokiStack and visualization
+
+The administrator role is any user who has permissions to deploy the operator and the cluster-wide resources required to deploy the logging components.
+
+### Goals
+
+* Drop support for the **ClusterLogging** custom resource
+* Drop support for **ElasticSearch**, **Kibana** custom resources and the **elasticsearch-operator**
+* Drop support for Fluentd collector implementation, Red Hat managed Elastic stack (e.g. Elasticsearch, Kibana)
+* Drop support in the **cluster-logging-operator** for **log-view-plugin** management
+* Support log forwarder API with minimal or no dependency upon reserved words (e.g. default)
+* Support an API to spec a Red Hat managed LokiStack with the logging tenancy model
+* Support an API to allow flexible deployment of the logging components: collector/forwarder, storage, visualization
+* Continue to allow deployment of a log forwarder to the output sinks of the administrators choosing
+* Automated migration path from *ClusterLogForwarder.logging.openshift.io/v1* to *ClusterLogForwarder.observability.openshift.io/v1*
```

Comment:
typo - v2

Reply:
I've struggled with the naming of the document and version.  It's the "v1" api of CLF.observability but its the next version of "cluster logging".  It really needs some clarification since "clusterlogging" as a resource will cease to exist

Reply:
Using `v1` under the new API version is good enough for me. We will start teaching people to use `clusterlogforwarder.observability.openshift.io` anyway. Let's not add confusion to them seeking/asking for where is `v1` if we release as `clusterlogforwarder.observability.openshift.io/v2`